README.md

```diff
@@ -0,0 +1,785 @@
+TopGit - A different patch queue manager
+
+
+Description
+===========
```

Comment:
Better to use `-` not `=`?  I think `=` is more commonly used for the document title, not for section titles.